ui/components/component-library/README.md

```diff
@@ -1,8 +1,38 @@
 # Component Library
 
-**⚠️ THESE COMPONENTS HAVE BREAKING CHANGES ⚠️**
+This folder contains design system components that are built 1:1 with the Figma [DS Components](https://www.figma.com/file/HKpPKij9V3TpsyMV1TpV7C/DS-Components?node-id=16%3A6&t=RFUEn25IPHUhqHVo-1) UI kit and should be used where possible in all UI feature work
```

Comment:
I see what you mean and totally agree. We do have these links in other internal docs but I like to connect things so that no matter where you are in the 'user journey' you have a link back to the source e.g. Figma -> Storybook, Storybook -> Figma. I'll rewrite this readme with the external contributor in mind and remove internal links. I'll also bring up the idea of making our Figma UI Kit publicly viewable. I think that's pretty cool and don't believe it creates any security or business risk IMO.

Reply:
@legobeat the link to the Figma file is now publicly viewable!

Reply:
Thank you sir! 🙏